docs/deployment.md

```diff
@@ -0,0 +1,56 @@
+# Deployment
+
+*This section is under construction*
```

Comment:
We can use the [admonition](http://squidfunk.github.io/mkdocs-material/extensions/admonition/), which is configurable with our theme to style this as a block.